src/Nethermind/Nethermind.HealthChecks.Test/ClHealthTrackerTests.cs

```diff
@@ -0,0 +1,53 @@
+// SPDX-FileCopyrightText: 2025 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+using FluentAssertions;
+using Nethermind.Core;
+using Nethermind.Logging;
+using NUnit.Framework;
+
+namespace Nethermind.HealthChecks.Test;
+
+public class ClHealthTrackerTests
+{
+    [Test]
+    public void ClHealthRequestsTracker_multiple_requests()
+    {
+        ManualTimestamper timestamper = new(DateTime.Parse("18:23:00"));
+        ClHealthRequestsTracker healthTracker = new(timestamper, 300, NullLogger.Instance);
```

Comment:
Missing dispose (surprisingly not detected by the IDE or compiler):

```suggestion
        await using ClHealthRequestsTracker healthTracker = new(timestamper, 300, NullLogger.Instance);
```